packages/gateway/src/features/registration/fhir-builders.ts

```diff
@@ -2078,15 +2088,17 @@ export const builders: IFieldBuilders = {
       }
       if (context.event === EVENT_TYPE.BIRTH) {
         if (fieldValue === 'MOTHER') {
-          await setInformantReference(
+          setInformantReference(
             MOTHER_CODE,
+            MOTHER_TITLE,
             relatedPersonResource,
             fhirBundle,
             context
           )
         } else if (fieldValue === 'FATHER') {
-          await setInformantReference(
+          setInformantReference(
             FATHER_CODE,
+            FATHER_TITLE,
```

Comment:
Can we add the 'GROOM' & 'BRIDE' cases here as well please?

Reply:
We also seems to have an `informantType` field that's the same as this relationship field. I wonder if both of them are needed or not?